JSONShootout.xcodeproj/project.pbxproj

```diff
@@ -73,6 +82,14 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		5BAB43C51DBE06A90017C770 /* Mapper.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Mapper.framework; path = "../../Library/Developer/Xcode/DerivedData/JSONShootout-bkocarfdpnsffvahaikdrlrmawsu/Build/Products/Debug-iphonesimulator/Mapper.framework"; sourceTree = "<group>"; };
+		5BAB43C61DBE06A90017C770 /* Marshal.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Marshal.framework; path = "../../Library/Developer/Xcode/DerivedData/JSONShootout-bkocarfdpnsffvahaikdrlrmawsu/Build/Products/Debug-iphonesimulator/Marshal.framework"; sourceTree = "<group>"; };
```

Comment:
Note that the existing frameworks have been changed to refer to user-specific derived data paths.